app/views/selfupdate_beta.scala.txt

```diff
@@ -171,11 +167,10 @@ fi
 
 # migrate deprecated sdkman_selfupdate_enable configuration
 @defining(LocalDate.now()) { now =>
-cp "$sdkman_config_file" "${sdkman_config_file}-@{now}.bak"
-
+  cp "$sdkman_config_file" "${sdkman_config_file}-@{now}.bak"
```

Comment:
This will always create a backup, regardless of if a migration is done. Was that your intention?

The reason I had an `if` statement was so that it would only happen when the migration was triggered.